src/ICU4N/Impl/Normalizer2Impl.cs

```diff
@@ -1557,73 +1557,71 @@ public bool GetCanonStartSet(int c, UnicodeSet set)
             return true;
         }
 
-        // ICU4N TODO: API - rename constants to follow .NET Conventions ?
-
         // Fixed norm16 values.
-        public const int MIN_YES_YES_WITH_CC = 0xfe02;
-        public const int JAMO_VT = 0xfe00;
-        public const int MIN_NORMAL_MAYBE_YES = 0xfc00;
-        public const int JAMO_L = 2;  // offset=1 hasCompBoundaryAfter=FALSE
-        public const int INERT = 1;  // offset=0 hasCompBoundaryAfter=TRUE
+        public const int MinYesYesWithCc = 0xfe02;
+        public const int JamoVt = 0xfe00;
+        public const int MinNormalMaybeYes = 0xfc00;
+        public const int JamoL = 2;  // offset=1 hasCompBoundaryAfter=FALSE
+        public const int Inert = 1;  // offset=0 hasCompBoundaryAfter=TRUE
 
         // norm16 bit 0 is comp-boundary-after.
-        public const int HAS_COMP_BOUNDARY_AFTER = 1;
-        public const int OFFSET_SHIFT = 1;
+        public const int HasCompBoundaryAfterValue = 1; // ICU4N-specific - added "Value" suffix to not conflict with method
+        public const int OffsetShift = 1;
 
         // For algorithmic one-way mappings, norm16 bits 2..1 indicate the
         // tccc (0, 1, >1) for quick FCC boundary-after tests.
-        public const int DELTA_TCCC_0 = 0;
-        public const int DELTA_TCCC_1 = 2;
-        public const int DELTA_TCCC_GT_1 = 4;
-        public const int DELTA_TCCC_MASK = 6;
-        public const int DELTA_SHIFT = 3;
+        public const int DeltaTccc0 = 0;
```

Comment:
I was looking through the standard, technical notes, and discussions online, and it seems like acronyms like "CCC" and "TCCC" are fairly common. Others like "FCD" stand for "Fast C or D" and I don't think those should be expanded. My recommendation would be to stick with the commonly-used acronyms, but we can expand some of the terms like "Comp." Thoughts?

Reply:
Given the discussion on the Impl namespace elsewhere, do we want to bother with this any further? IMO it seems like keeping the names closer to the original ICU4J will ease porting code that depends on it, although it seems like this project will soon discourage people from depending on these types/members.

Reply:
Yeah, I was thinking about this a bit more, and since this is an "advanced" API, I suppose using this approach is probably fine. And it keeps it more inline with the upstream code.